resources/gemini-1b/variables.tf

```diff
@@ -19,5 +19,11 @@ variable "droplets-per-region" {
 variable "node-snapshot-tag" {
   description = "Node snapshot tag"
   type = string
-  default = "gemini-1b-2022-june-05"
+  default = "gemini-1b-2022-jun-10"
```

Comment:
Should there be a newer version ? gemini-1b-2022-jun-18 

https://github.com/subspace/subspace/pkgs/container/node/26000223?tag=gemini-1b-2022-jun-18

Reply:
good catch. Will need to redeloy those